### PR TITLE
Added Django for Everybody and Restructuring of Python topic

### PR DESCRIPTION
**Pull request by BasilVictor**
## Hacktoberfest notes:

- due to volume of submissions, we may not be able to review PRs that do not pass tests and do not have informative titles.
- please read our [contributing guidelines](/CONTRIBUTING.md)
- be sure to check the output of Travis-CI for linter errors
- if this is your first open source contribution, make sure it's not your last!

## What does this PR do?
Add Resource(s) , Improve Repo

### Description 
Add Resource(s) - Added Django for Everybody (by Charles R. Severance) open source materials of the Coursera course
Improve Repo - Restructured Python category and added Django and Flask as sub topics

### Why is this valuable (or not)?
It is a great place for Django beginners

### How do we know it's really free?
It is a `Copyright Creative Commons Attribution 3.0` website.

### For book lists, is it a book? For course lists, is it a course? etc.
It is a course

image

### Checklist:
- [x] Not a duplicate
- [x] Included author(s) if appropriate
- [x] Lists are in alphabetical order
- [x] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

File: free-courses-en.md [hunk 1]
```diff
@@ -464,6 +463,17 @@
 * [The Python Tutorial](https://docs.python.org/3/tutorial/)
 
 
+#### Django
+
+* [Django for Everybody](https://dj4e.sites.uofmhosting.net/lessons) (Coursera Reading Materials with YouTube Videos)
+* [Python Django Basics Tutorial](https://www.youtube.com/playlist?list=PL-osiE80TeTtoQCKZ03TU5fNfx2UY6U4p) - Corey Schafer
+
+
+#### Flask
```
Comment (SethiPandi) [inline free-courses-en.md:472]: There should be a line break under here.

File: free-courses-en.md [hunk 2]
```diff
@@ -464,6 +463,17 @@
 * [The Python Tutorial](https://docs.python.org/3/tutorial/)
 
 
+#### Django
+
+* [Django for Everybody](https://dj4e.sites.uofmhosting.net/lessons) (Coursera Reading Materials with YouTube Videos)
```
Comment (SethiPandi) [inline free-courses-en.md:468]: Please keep lists in alphabetical order.
D is before F.

**Review by SethiPandi — CHANGES_REQUESTED** [review]
I left some comments for you that should hopefully help address the formatting issues.

You can check out the contribution guide (CONTRIBUTING.md) for more information on formatting and other guidelines for this repository. You can click on the failed check below and head to Travis CI to see exactly what was wrong with your PR and how to fix it!

Edit: I originally left two comments, but deleted one as it was a non-issue as I simply miss read your change. Sorry about that!

**Comment by BasilVictor** [comment]
@SethiPandi  resolved

**Comment by eshellman** [comment]
Thanks! Opinions on #4573 welcome.

**Review by PawaN-kingS — APPROVED** [review]

**Pull request merged**